lib/platform/github/index.ts

```diff
@@ -1719,3 +1725,60 @@ export async function getVulnerabilityAlerts(): Promise<VulnerabilityAlert[]> {
   }
   return alerts;
 }
+
+async function pushFiles(
+  { branchName, message }: CommitFilesConfig,
+  { parentCommitSha, commitSha }: CommitResult
+): Promise<CommitSha | null> {
+  try {
+    await pushCommitToRenovateRef(commitSha, branchName);
+    const treeItems = await listCommitTree(commitSha);
+
+    const treeRes = await githubApi.postJson<{ sha: string }>(
+      `/repos/${config.repository}/git/trees`,
+      { body: { tree: treeItems } }
+    );
+    const treeSha = treeRes.body.sha;
+
+    const commitRes = await githubApi.postJson<{ sha: string }>(
+      `/repos/${config.repository}/git/commits`,
+      { body: { message, tree: treeSha, parents: [parentCommitSha] } }
+    );
+    const remoteCommitSha = commitRes.body.sha;
+
+    if (git.branchExists(branchName)) {
+      await githubApi.patchJson(
```

Comment:
```suggestion
      // This is the equivalent of a git force push
      // We are using this REST API because the GraphQL API doesn't support force push
      await githubApi.patchJson(
```